src/properties.ts

```diff
@@ -152,6 +154,16 @@ export const properties = {
     default: ".syntest/instrumented",
   },
 
+  // plugins
+  use_plugin: {
```

Comment:
Can this be called multiple times? Or is the idea that the user provides a list of plugins? In the latter would a list format not be better?

Reply:
In the cli you use: 
--use-plugin plugin/A plugin/B
or:
--use-plugin plugin/A --use-plugin plugin/B

In the config its: 
use_plugin: ["plugin/A", "plugin/B"]

Does this answer your question?

Reply:
Yes